test/commands/extended/send_transfer_test.py

```diff
@@ -163,6 +167,7 @@ def test_pass_optional_parameters_omitted(self):
       {
         'changeAddress':      None,
         'inputs':             None,
+        'reference':          None,
```

Comment:
❔ Good thinking!  Could we also include unit tests for invalid values for `reference` (i.e., not TrytesCompatible)?